src/DB.php

```diff
@@ -872,7 +872,7 @@ public static function insertMessageRequest(Message $message)
             $sth->bindValue(':forward_date', $forward_date);
 
             $reply_to_chat_id = null;
-            if ($reply_to_message_id) {
+            if ($reply_to_message_id !== null) {
```

Comment:
`!is_null()`

Reply:
@Pub4Game Welcome to the "every millisecond counts" game 😹 
`is_null()` is slower, as it adds a function call.

Using PHP5, it can be up to 10x slower.
Look here and give the code example a try:
https://hakre.wordpress.com/2011/03/29/php-is_null-vs-null/

PHP7+ fixes this, making `is_null()` a bit quicker even sometimes.

Reply:
I think we live in a time when php5 needs to go out of our lives... 😃

Reply:
Haha, for sure! When 7.2 is officially out, i really hope more people start dropping 5.